src/panels/config/integrations/dialog-add-integration.ts

```diff
@@ -78,8 +79,9 @@ class AddIntegrationDialog extends LitElement {
 
   private _height?: number;
 
-  public showDialog(params): void {
+  public showDialog(params: AddIntegrationDialogParams): void {
     this._open = true;
+    this._pickedBrand = params.brand;
```

Comment:
You could just store params

Reply:
But it used outside of params too? Just want to set it as initial value?